DietPi-Prep | RPi: Fix "--reinstall" not marking as "manual"

MichaIng opened the pull request:
Just tested script on my RPi. Worked well besides: apt install --reinstall does not change apt-mark flag. Thus RPi specific packages get autoremoved afterwards. The package unhold/rm modules/--reinstall was just for own to package kernel transition and should be not necessary anymore, right?

Commit:
v6.2

+ DietPi-Prep | Remove RPi kernel transition parts, fix --reinstall not marking as "manual" installed

Commit:
v6.2

+ Allow RPi buster upgrade by skipping incompatible 'raspi-copies-and-fills'.

Fourdee commented:
@MichaIng 

> The package unhold/rm modules/--reinstall was just for own to package kernel transition and should be not necessary anymore, right?

This was basically to achieve fresh update/reinstall of RPI kernel via APT:
- Unhold APT kernel, just in case user (or us) is using ```rpi-update``` and has held packages.
- To clear all modules (to be replaced with either newer versions, or reinstall). APT/rpi-update does not do this.

Effectively, its a "cover all" method to ensure only APT kernel is installed, regardless of rpi-update etc.

MichaIng commented:
@Fourdee
Okay in that case (we want to assure just APT firmware) I will revert changes, just adding G_AGI without --reinstall to the front. This ensures all firmware packages are really installed and marked as manual installed, as --reinstall does not install missing packages and does not add mark manual, leaving the image without any firmware.

€: Done, ups v6.0 typo 😆.

Commit:
v6.0

+ DietPi-Prep | RPi: Remove and force reinstall of firmware to assure APT package firmware only

Commit:
v6.2

+ DietPi-Prep | RPi: Unhold firmware APT packages, just in case

Commit:
v6.2

+ DietPi-Prep | RPi: "raspi-copies-and-fills" and "raspberrypi-sys-mods" are not part of firmware, thus don't need to be reinstalled

Fourdee reviewed (approved):
👍

Fourdee commented:
@MichaIng 

Legend, thank you 👍
